monkey/monkey_island/cc/server_utils/file_utils.py

```diff
@@ -21,6 +21,12 @@ def is_windows_os() -> bool:
     import monkey_island.cc.server_utils.windows_permissions as windows_permissions
 
 
+def get_file_contents(file_path: Path) -> str:
```

Comment:
This doesn't read files in binary mode.
```suggestion
def get_text_file_contents(file_path: Path) -> str:
```

Maybe `read` instead of `get`, but it's up to you.
```suggestion
def read_text_file_contents(file_path: Path) -> str:
```